examples/4337-passkeys/README.md

```diff
@@ -1,6 +1,6 @@
 # Safe + 4337 + Passkeys example application
```

Comment:
This looks like code changes, but it is just how GitHub renders it. In fact, it is the code from the `4337-passkeys-singleton-signer` that just overwrote the deprecated `4337-passkeys` example).